packages/kolibri-common/components/ExtraDemographics/index.vue

```diff
@@ -41,7 +41,7 @@
     },
     computed: {
       customSchema() {
-        return this.facilityDatasetExtraFields.demographic_fields || [];
+        return this.facilityDatasetExtraFields?.demographic_fields || [];
```

Comment:
If we allow `facilityDatasetExtraFields` to be null. Then this prop should not be marked as required right? https://github.com/rtibbles/kolibri/blob/787162af96d4af31dc23ec06d370692524679c42/packages/kolibri-common/components/ExtraDemographics/index.vue#L35

Reply:
I mean, it is required - it's just that VueJS prop validation doesn't actually do anything in production, it's advisory to ensure that developers are using props appropriately. The issue arises because a value _is_ passed in for the prop, but the value is `null`.

Reply:
Oh, well that makes sense haha, thanks!